plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -129,6 +130,16 @@ public void setUp()
                 .maximumSize(1000)
                 .partitionCacheEnabled(true)
                 .build();
+        // only cache stats
+        statsCacheMetastore = CachingHiveMetastore.builder()
+                .delegate(new BridgingHiveMetastore(thriftHiveMetastore))
+                .executor(executor)
+                .cacheTtl(new Duration(5, TimeUnit.MINUTES))
+                .refreshInterval(new Duration(1, TimeUnit.MINUTES))
+                .maximumSize(1000)
+                .partitionCacheEnabled(true)
+                .metadataCacheEnabled(false)
```

Comment:
reoder

use explicit statsCacheEnabled(true), since you explicitly want that (`// only cache stats`)